controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -52,10 +47,24 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
+
+import io.pravega.shared.protocol.netty.WireCommands;
```

Comment:
Our style guide dictates that imports are to be in alphabetical order. Followed by static imports in alphabetical order. You can configure your IDE to do this for you. Please do that and have it re-order these.